thunder/clang/__init__.py

```diff
@@ -2042,3 +2042,15 @@ def argsort(a: TensorProxy, /, dim: None | int = None, descending: bool = False,
     dim = utils.canonicalize_dim(a.ndim, dim)
 
     return prims.argsort(a, dim, descending, stable)
+
+
+@clangop()
+def _grouped_mm(
+    a: TensorProxy,
+    b: TensorProxy,
+    offsets: TensorProxy = None,  # Unused in meta, but type-checked
+    bias: TensorProxy = None,
+    dtype=None,
```

Comment:
nit-picking: type annotation. note that I've not checked how `dtypes` is imported
```suggestion
    offsets: TensorProxy | None = None,  # Unused in meta, but type-checked
    bias: TensorProxy | None = None,
    dtype: dtypes.dtype | None = None,
```